input/pagecontent/4-6-fhircast-event-content-sharing.md

```diff
@@ -31,8 +31,7 @@ Using a reporting application, a clinical user creates a report by choosing an i
   {% include ContentExchangeOpenReport.svg %}
   <figcaption><b>Figure: Opening a Diagnosticreport Context</b></figcaption><p></p>
 </figure>
-
-The clinical user takes a measurement using the imaging reading application (message 1 in the below sequence diagram) which then shares this measurement by making a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) request to the Hub (message 2). The Hub validates that the `context.versionId` provided in the request is correct, updates its content, generates a new `context.versionId` (message 3). If the `context.versionId` provided in the request is not correct the Hub rejects the request (response to message 2). The Hub then distributes `DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) events which contain the newly generated `context.versionId` and the `priorVersionId` to all subscribed applications (messages 4a, 4b, and 4c). The reporting application receives the measurement through a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) event from the Hub and adds this information to the report if the `context.versionId` it currently holds matches the `context.priorVersionId` provided in the event (message 7). If the `context.priorVersionId` does not match the `context.versionId` of the content known to the reporting application, it may resynchronize its content by requesting the current context from the Hub (message 8).
+After opening a report, the clinical user takes a measurement using the imaging reading application (message 1 in the below sequence diagram) which then shares this measurements by making a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) request to the hub (message 2). The Hub validates that the `context.versionId` provided in the request is correct, updates its content, generates a new `context.versionId` (message 3). If the `context.versionId` provided in the request is not correct the Hub rejects the request (response to message 2). The Hub then distributes `DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) events which contain the newly generated `context.versionId` and the `priorVersionId` to all subscribed applications (messages 4a, 4b, and 4c). The reporting application receives the measurement through a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) event from the Hub and adds this information to the report if the `context.versionId` it currently holds matches the `context.priorVersionId` provided in the event (message 7). If the `context.priorVersionId` does not match the `context.versionId` of the content known to the reporting application, it may resynchronize its content by requesting the current context from the Hub (message 8).
```

Comment:
```suggestion
After opening a report, the clinical user takes a measurement using the imaging reading application (message 1 in the below sequence diagram) which then shares this measurement by making a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) request to the hub (message 2). The Hub validates that the `context.versionId` provided in the request is correct, updates its content, generates a new `context.versionId` (message 3). If the `context.versionId` provided in the request is not correct the Hub rejects the request (response to message 2). The Hub then distributes `DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) events which contain the newly generated `context.versionId` and the `priorVersionId` to all subscribed applications (messages 4a, 4b, and 4c). The reporting application receives the measurement through a [`DiagnosticReport-update`](3-6-3-DiagnosticReport-update.html) event from the Hub and adds this information to the report if the `context.versionId` it currently holds matches the `context.priorVersionId` provided in the event (message 7). If the `context.priorVersionId` does not match the `context.versionId` of the content known to the reporting application, it may resynchronize its content by requesting the current context from the Hub (message 8).
```